src/app.js

```diff
@@ -78,6 +78,7 @@ var app = function(options) {
          options.env == 'production',       'env must be production or development');
   assert(options.forceSSL !== undefined,    'forceSSL must be defined');
   assert(options.trustProxy !== undefined,  'trustProxy must be defined');
+  assert(!options.rootDocsRedirect || options.docs, 'options.docs must be given if rootDocsRedirect is specified');
```

Comment:
This is adding two new options -- they should be documented above (around line 64) and in the README.

Reply:
Also, I'm not sure this code is right (some tests might help).

First, `rootDocsRedirect` defaults to undefined (which is false), but we would like it to default to true.

Here's what we expect:
* `App({rootDocsRedirect: true, docs: Docs})` - OK
* `App({rootDocsRedirect: false, docs: Docs})` - OK
* `App({rootDocsRedirect: true})` - ERROR
* `App({rootDocsRedirect: false})` - OK

but that doesn't match what this assert says.

Reply:
but since we are sending `rootDocsRedirect : true,` from tc-hooks , it will give an error only when docs is not defined.

Reply:
That's true, but what about other applications where rootDocsRedirect is false?

Reply:
Error is generated only in the third case according to the assert also(as we expect).

Reply:
Adding `rootDocsRedirect: true,` around line 76 would solve the problem . Right?

Reply:
You're right, I had my logic backward before.  The assert is correct -- but we still need the default!  Yes, the `_.defaults` call is the right place to add a default.